docs/react-testing-library/setup.md

```diff
@@ -144,6 +144,35 @@ module.exports = {
 
 </details>
 
+<details>
+<summary>Modify queries globally with custom render</summary>
```

Comment:
```suggestion
<summary>Customize queries globally with custom render</summary>
```

Reply:
Will update! Even if my British instincts make me want to correct it to 'Customi**s**e' 😂